uav_names: ["uav1", "uav2", "uav3", "uav4", "uav5"] #, "uav6", "uav7", "uav8"]

# odometry_topic: "estimation_manager/ground_truth/odom"
odometry_topic: "estimation_manager/odom_main"

set_reference_timer:
  rate: 10 # [Hz]

diagnostics:
  rate: 0.5 # [Hz]
  odom_timeout: 1.0 # [s]

odom_msg_max_latency: 1.0 # [s]

# control approach params
controlled_dimensions: 3


d1: 0.2
d2: 1
d3: 0.2
d4: 1
radius: 3
encumbrance: 2.3
betaD: 1.0
beta_min: 0.5
dt: 0.05
step_size: 0.05
maximum_distance_conn: 17.0

